astro.config.mjs

```diff
@@ -236,6 +236,9 @@ const config = defineConfig({
         ]
       })
     ]
+  },
+  redirects: {
+    "/enterprise/authentication/": "/enterprise/privacy/"
```

Comment:
Vercel already handles this, so there's no need to create a redirect at this level